### PR TITLE
[IOTDB-922] Int and Long can convert to each other in ResultSet

### PR DESCRIPTION
**Pull request by yuqi1129**

### CODE REVIEW

File: server/src/test/java/org/apache/iotdb/db/integration/IoTDBTestResultSet.java [hunk 1]
```diff
@@ -0,0 +1,105 @@
+package org.apache.iotdb.db.integration;
+
+import com.clearspring.analytics.util.Lists;
```
Comment (HTHou) [inline server/src/test/java/org/apache/iotdb/db/integration/IoTDBTestResultSet.java:3]: Missing apache header, please take a look.

File: server/src/test/java/org/apache/iotdb/db/integration/IoTDBTestResultSet.java [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+/**
+ * @author yuqi
+ * @mail yuqi5@xiaomi.com
+ * @description your description
+ * @time 27/9/20 22:56
+ **/
+public class IoTDBTestResultSet {
+    private static List<String> sqls = Lists.newArrayList();
+    private static Connection connection;
+
```
Comment (HTHou) [inline server/src/test/java/org/apache/iotdb/db/integration/IoTDBTestResultSet.java:28]:
```suggestion
public class IoTDBTestResultSet {
  private static List<String> sqls = Lists.newArrayList();
  private static Connection connection;

```
Actually, we use double space symbols to intend the code.

File: server/src/test/java/org/apache/iotdb/db/integration/IoTDBTestResultSet.java (hunk 1 above)
Comment (jixuan1989) [reply server/src/test/java/org/apache/iotdb/db/integration/IoTDBTestResultSet.java:3]: and better to changethe name to IoTDBResultSetIT

**Comment by jixuan1989** [comment]
this PR... has modified about 1900 files?

**Comment by yuqi1129** [comment]
This is the seconds time i met this problem, the same happened yesterday,
i will update the mr now

Xiangdong Huang <notifications@github.com> 于2020年9月29日周二 上午11:54写道：

File: server/src/test/java/org/apache/iotdb/db/integration/IoTDBTestResultSet.java (hunk 1 above)
Comment (yuqi1129) [reply server/src/test/java/org/apache/iotdb/db/integration/IoTDBTestResultSet.java]: OK

File: server/src/test/java/org/apache/iotdb/db/integration/IoTDBTestResultSet.java (hunk 2 above)
Comment (yuqi1129) [reply server/src/test/java/org/apache/iotdb/db/integration/IoTDBTestResultSet.java]: Right

**Comment by yuqi1129** [comment]
All has resolved

**Review by qiaojialin — APPROVED** [review]

**Review by HTHou — APPROVED** [review]

**Pull request merged**